github/PaginatedList.py

```diff
@@ -175,7 +175,7 @@ def _fetchNextPage(self):
         self.__nextParams = None
 
         if self.__list_item in data:
-            self.__totalCount = data['total_count']
+            self.__totalCount = data.get('total_count')
```

Comment:
How does this change relate to the other one?

Reply:
I'm not sure why you decided that this key exists if the if self.__list_item in data. in this case it doesn't, so I'm using .get to not fail, and it works fine (anyway it's None by default).

Reply:
I see. Then, to be consistent, the same change should also be made to the `get_page` method.